src/firefly/js/core/background/JobInfo.jsx

```diff
@@ -88,9 +89,14 @@ function DataOrigin({dataOrigin, type, jobId}) {
     );
 }
 
-function FinalMsg({phase, summary, error}) {
+function FinalMsg({phase, summary, error, tbl_id}) {
     if (phase === 'COMPLETED') {
-        return <KeywordBlock label='Summary' value={summary}/>;
+        return (
+            <Stack direction='row' spacing={1} alignItems='center'>
+                <KeywordBlock label='Summary' value={summary}/>
+                <OverflowMarker tbl_id={tbl_id}/>
```

Comment:
Relevant to #1542 (fix-up PR for this feature): In the table-info dialog I wanted the overflow "warning triangle" marker to be followed by the words "Result truncated".